src/FantasyFun.DAL/Repositories/LeaguesRegistration.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```

Comment:
Almost all files has unnecessary namespaces. It will be good if some day we fix that but without R# (Resharper) it is not easy to do that very quick. So for now, we leave it as it is, but for the future try to control namespaces.